us.materialio.Materialious.yml

```diff
@@ -0,0 +1,49 @@
+app-id: us.materialio.Materialious
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
```

Comment:
this is the wrong version, it MUST match the runtime.

Reply:
Ty, fixed!